code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -2973,6 +2992,24 @@
 	reagents.add_reagent(TOMATOJUICE, 6)
 	bitesize = 2
 
+/obj/item/weapon/reagent_containers/food/snacks/sliceable/pizza/meatpizza/make_poisonous()
+	reagents.clear_reagents()
+
+	var/datum/disease2/disease/new_virus = new /datum/disease2/disease
+	new_virus.makerandom()
+
+	var/list/blood_data = list(
+		"donor" = null,
+		"viruses" = null,
+		"blood_DNA" = null,
+		"blood_type" = "AB+",
+		"resistances" = null,
+		"trace_chem" = null,
+		"virus2" = list()
+	)
+	blood_data["virus2"]["[new_virus.uniqueID]"] = new_virus
+	reagents.add_reagent(BLOOD, 56, blood_data)
```

Comment:
56?

Reply:
What might appear to be a magic number is in fact the amount of reagents in the original pizza. I'll make it clearer.

Reply:
I see... *define when*?